description: forward declare method that is defined after it is used
author: Michael Gilbert <mgilbert@debian.org>

--- a/third_party/WebKit/Source/platform/wtf/LinkedHashSet.h
+++ b/third_party/WebKit/Source/platform/wtf/LinkedHashSet.h
@@ -685,6 +685,8 @@ inline LinkedHashSet<T, U, V, W>& Linked
   return *this;
 }
 
+inline void SwapAnchor(LinkedHashSetNodeBase& a, LinkedHashSetNodeBase& b);
+
 template <typename T, typename U, typename V, typename W>
 inline void LinkedHashSet<T, U, V, W>::Swap(LinkedHashSet& other) {
   impl_.swap(other.impl_);
